kolibri/utils/version.py

```diff
@@ -38,40 +38,32 @@
-|              |                     |                     |                           | Changes                             |
+|              |                     |                     |                           | 5 changes                           |
 |              |                     |                     |                           | since tag:                          |
-|              |                     |                     |                           | 1.2.3b1.dev123-f1234567             |
+|              |                     |                     |                           | 1.2.3b1.dev+git.5.f1234567          |
 +--------------+---------------------+---------------------+---------------------------+-------------------------------------+
 | rc1+         | (1, 2, 3, 'alpha',  | Fallback            | ``git describe --tags``   | Clean head:                         |
 | (release     | 1)                  |                     |                           | 1.2.3rc1,                           |
 | candidate)   |                     |                     |                           | Changes                             |
 |              |                     |                     |                           | since tag:                          |
 |              |                     |                     |                           | 1.2.3rc1.dev123-f1234567            |
 +--------------+---------------------+---------------------+---------------------------+-------------------------------------+
-| beta0, rc0,  | Not recommended,    | Fallback            | timestamp of latest       | 0.4.0b0.dev020170605181124-f1234567 |
-| post0, x.y.0 | but if you use it,  |                     | commit + hash             |                                     |
-|              | your release        |                     |                           |                                     |
-|              | transforms into a   |                     |                           |                                     |
-|              | X.Y.0b0.dev{suffix} |                     |                           |                                     |
-|              | release, which in   |                     |                           |                                     |
-|              | most cases should   |                     |                           |                                     |
-|              | be assigned to the  |                     |                           |                                     |
-|              | preceding release   |                     |                           |                                     |
-|              | type.               |                     |                           |                                     |
+| beta0, rc0,  | DO NOT USE          | Fallback            | timestamp of latest       | 1.2.3b0.dev+git.123.f1234567        |
```

Comment:
This was added for clarity. Let's not use these sort of weird and undefined version numbers ever :)